Makefile

```diff
@@ -4,10 +4,10 @@
 # You can set these variables from the command line.
 SPHINXBINS    = .tox/docs/bin
 SRCDIR        = source
-SPHINXOPTS    = #-W
+SPHINXOPTS    = # used by html_languages to specify $$lang
 SPHINXBUILD   = sphinx-build
 PAPER         =
-BUILDDIR      = _build
+BUILDDIR      = target
```

Comment:
This may be a mistake, but something here is inconsistent:

* `_build` actually contains committed `_build/locale/` files from two years ago... 
* `make clean` is willing to remove this folder
* The recursive `html_languages` target overrides this to `target/doc/$$lang`

Conclusion is that `target` is correct, and in normal use `target/doc/en` and `target/doc/fr` are used.

I think we could remove `_build/locale/` files?

Reply:
@juanluisrp is the above understanding correct? should I remove `_build/locale` files?

Reply:
I have answered my own question, translations are handled in:

* `source/locale`
* `translations`

Both of which are mentioned in `.gitignore`.